next/components/media--file-attachments.tsx

```diff
@@ -0,0 +1,43 @@
+import { absoluteUrl } from "@/lib/utils";
+import { formatFileSizeInBytes } from "@/lib/utils";
+import { FileAttachments } from "@/lib/zod/paragraph";
+import ListIcon from "@/styles/icons/list.svg";
+
+interface MediaFileAttachmentsProps {
+  mediaItems: FileAttachments["field_file_attachments"];
+}
+
+export function MediaFileAttachments({
+  mediaItems,
+  ...props
+}: MediaFileAttachmentsProps) {
+  if (mediaItems.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      <ul {...props} className="list-inside space-y-2">
+        {mediaItems.map((mediaItem) => (
```

Comment:
I built this simple listing view with the info we have for the file. A more advanced version would be to have icons for different file types, and a fallback icon for when they are not recognized.

I wonder if we should just go ahead and add this more "advanced" version in the template?

Reply:
I made a new jira ticket for this, let's go with the current solution for now.